src/components/App/App.js

```diff
@@ -34,11 +28,7 @@ class Home extends Component {
 
     const approxStoryHeight = 70;
```

Comment:
if we want to get crazy, we can use a `ref` on the first list item and get it's height and then add it to the Redux store. Probably overkill though 😅